pennylane/pauli/conversion.py

```diff
@@ -367,7 +367,7 @@ def is_pauli_sentence(op):
     """Returns True of the operator is a PauliSentence and False otherwise."""
     if op.pauli_rep is not None:
         return True
-    if isinstance(op, (Hamiltonian, LinearCombination)):
+    if isinstance(op, Hamiltonian):
         return all(is_pauli_word(o) for o in op.ops)
```

Comment:
`Hamiltonian` now has a Pauli rep, so the first if statement will return `True` before we reach the second if statement. The second if statement isn't needed. Can't add a code suggestion for some reason.